modules/ding_config_import/README.md

```diff
@@ -0,0 +1,74 @@
+# Ding configuration import
+This module can be used to import configuration changes into the site and enable/disable modules.
+
+The configuration is described by using YAML files with arrays that are flattened to variable keys
+if they are associative arrays. If "normal" arrays they are written as arrays to variables.
+
+If an associative array contains the stop key `array: 1` the array it is placed in is not flattened
+thereby allow for complex variable configuration.
```

Comment:
:+1: for this. Perhaps consider prefixing the key with a special character to make it stand out more? E.g. `_array`. You might also consider using [a proper boolean value](https://yaml.org/type/bool.html) instead of a numeric value.